pkg/skaffold/deploy/helm/helm.go

```diff
@@ -260,40 +262,66 @@ func (h *Deployer) Deploy(ctx context.Context, out io.Writer, builds []graph.Art
 
 	nsMap := map[string]struct{}{}
 	manifests := manifest.ManifestList{}
+	g, ctx := errgroup.WithContext(ctx)
 
+	if h.Concurrency != nil && *h.Concurrency == 1 {
```

Comment:
Would you be able to add tests for both sequential and concurrent cases?

Reply:
hi @alphanota, pushed changes. by default it keeps the old behavior (sequential order), but a user can change it using the config file.  
Please check this PR as well https://github.com/GoogleContainerTools/skaffold/pull/9520, because it's very important, if something goes wrong with helm package installation, user will have package in `pending` state, because skaffold kills the helm command, so the user needs to manually uninstall/rollback the pending release